docs/60_Usage_Best_Practices.md

```diff
@@ -268,26 +268,7 @@ These data can be dynamic or static, but in the case of dynamic data, it is impo
 
 When the dataset of a collection has a high rate of **UPDATE/DELETE** operations compared to the rate of incoming HTTP requests, we suggest avoiding pagination mechanisms in favor of a data streaming approach.
 
-The `GET /export` method exposed by each endpoint associated with a collection opens a data stream in different formats. By using this method, the CRUD Service will open **only one cursor** to the MongoDB cluster, and the `ResultSet` will remain unaffected by concurrent **UPDATE/DELETE** operations.
-
-The export format can be specified through the `Accept` header, supported formats (and relative headers) are the following ones:
-
-
-| Format | Accept Header value | Description |
-|--------|---------------------|-------------|
-| `json` | `application/json` | Data is exported in JSON format. |
-| [`ndjson`](https://en.wikipedia.org/wiki/JSON_streaming#Newline-delimited_JSON) | `application/x-ndjson` | Data is exported in JSON, each record is processed individually and separated by a newline (`\n`) delimiter. |
-| `csv` | `text/csv` | Data is exported in CSV format using comma as separator (the CSV includes the header with column names) |
-| `xlsx` | `application/vnd.openxmlformats-officedocument.spreadsheetml.sheet` | Data is exported in XLS format (the file includes the header with column names) |
-| `xls` | `application/vnd.ms-excel` | Data is exported in XLS format (the file includes the header with column names) |
-
-In the given scenario, we can make a single HTTP request:
-
-- `GET /my_single_view/export`
```

Comment:
I would recommend leaving these lines here (from 284 onward), as they are part of the guide.